native_client/ctcdecode/scorer.h

```diff
@@ -78,35 +80,25 @@ class Scorer {
   void *dictionary;
 
 protected:
-  // necessary setup: load language model, set char map, fill FST's dictionary
-  void setup(const std::string &lm_path,
-             const std::vector<std::string> &vocab_list);
+  // necessary setup: load language model, fill FST's dictionary
+  void setup(const std::string &lm_path, const std::string &trie_path);
 
   // load language model from given path
   void load_lm(const std::string &lm_path);
 
   // fill dictionary for FST
-  void fill_dictionary(bool add_space);
-
-  // set char map
-  void set_char_map(const std::vector<std::string> &char_list);
+  void fill_dictionary(const std::vector<std::string> &vocabulary, bool add_space);
 
   double get_log_prob(const std::vector<std::string> &words);
 
-  // translate the vector in index to string
-  std::string vec2str(const std::vector<int> &input);
-
 public:
   void *language_model_;
```

Comment:
Know it's not your code, but `void*` in C++, come on.

Reply:
I guess they were going to use the static KenLM interface but then switched to the virtual one and forgot to update this. Will fix.